AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClientExtensions.swift

```diff
@@ -172,10 +172,13 @@ extension AWSMobileClient {
     ///   - username: username of the user.
     ///   - password: password of the user.
     ///   - validationData: validation data for this sign in.
```

Comment:
Should we mark here that validationData is ignored if clientMetaData is provided.

Reply:
There's a slight improvement in this implementation, we always set the `clientMetaData` is exist, then replaces the value if there is a key from `validationData`, so technically we're not ignoring the clientMetadata, but validatonData takes precedent (if developers use both validationData and clientMetadata, the data in validationData will override clientMetadata). adding some comment in code